Make the Version Numbers for 1.0.2 identical

AnHardt opened the pull request:
This is the minimal possible change.
Going over the Configuration.h would require to change them all.

thinkyhead committed:
Patch to make Z look more like X and Y on UltraLCD

thinkyhead committed:
Patch to make Z look more like X and Y on UltraLCD

thinkyhead committed:
Least stack-usage self-contained ftostr32np()

This is the optimal code for a self-contained formatter, although the
original code is crafty in being smaller and simpler, and can be
evaluated as using the original output as a scratch pad for state,
making the final formatter more straightforward. While this code is
longer, all code-paths are minimal.

thinkyhead committed:
Least stack-usage self-contained ftostr32np()

This is the optimal code for a self-contained formatter, although the
original code is crafty in being smaller and simpler, and can be
evaluated as using the original output as a scratch pad for state,
making the final formatter more straightforward. While this code is
longer, all code-paths are minimal.

thinkyhead committed:
Least stack-usage self-contained ftostr32np()

This is the optimal code for a self-contained formatter, although the
original code is crafty in being smaller and simpler, and can be
evaluated as using the original output as a scratch pad for state,
making the final formatter more straightforward. While this code is
longer, all code-paths are minimal.

thinkyhead committed:
Least stack-usage self-contained ftostr32np()

This is the optimal code for a self-contained formatter, although the
original code is crafty in being smaller and simpler, and can be
evaluated as using the original output as a scratch pad for state,
making the final formatter more straightforward. While this code is
longer, all code-paths are minimal.

RobotCaleb committed:
Language change implementation revamped. Didn't work as intended with Arduino IDE.

RobotCaleb committed:
Fix spacing

RobotCaleb committed:
Fix spacing

thinkyhead committed:
As it should be

thinkyhead committed:
As it should be

thinkyhead committed:
ftostr32np > ftostr32sp

sp=space-padded, similar to: ns=no-sign

thinkyhead committed:
ftostr32np > ftostr32sp

sp=space-padded, similar to: ns=no-sign

boelle committed:
Delete .travis.yml

boelle committed:
Delete .gitignore

boelle committed:
Update README.md

fmalpartida committed:
Merge branch 'SAV-MkI_merge' into Development

Tested for SAV MKI and SAV 3D LCD on lewihe.

wgm4321 committed:
Add Filament menu and add Filament/Retract settings to config store.

wgm4321 committed:
Add Filament menu and add Filament/Retract settings to config store.

wgm4321 committed:
Fix syntax error caused by previous changes.

wgm4321 committed:
Fix syntax error caused by previous changes.

tonnico committed:
change filament issue

make a more general solution

tonnico committed:
Rename ignore_blocks_queued

more speaking name

tonnico committed:
Expand manage_inactivity #1264

void manage_inactivity(bool ignore_stepper_queue=false)

standard is false so it is not necessary to change additional code. this
change should be better, hopefully nophead like this :)

bkubicek committed:
Update Marlin_main.cpp

fixed minor typo

bkubicek committed:
Addes servo0 pin ultimaker

RobotCaleb committed:
Delta calibration menu based on http://minow.blogspot.com/index.html#4918805519571907051
To test/enable, uncomment the #define DELTA_CALIBRATION_MENU in Configuration.h

RobotCaleb committed:
Actual menu change. Moved menu logic to not be shown unless not currently printing. (untested but straight-forward, printer in a bad state currently)

RobotCaleb committed:
Using the new language implementation wasn't behaving as expected.

No matter what LANGUAGE_CHOICE was set to it would always take the en branch.
I revamped it a bit to use a macro to create the language include file name. This required a slight tweak to the RU declaration since it's special-cased in a couple places in code.

This is an attempt to pull my changes from #1222 over.

RobotCaleb committed:
Move delta calibration menu to sample delta configuration.
Turn off by default.

Commit:
fixed compilation

bkubicek committed:
Update Marlin.h

thinkyhead committed:
Fix regressions affecting compilation

This fixes two regressions, caused by #1167 and #1191 .

Grogyan committed:
Update pins.h

Somehow the pin definitions for the MAX6675 Slave Select/Chip Select got changed back to the hardware SPI SS/CS.  
Pin 49 cannot be used as that is used by the SD card socket to detect if an SD card is present.
Pin 53 is the Hardware SPI SS, which is dedicated, nicely, to the SD card SS and therefore cannot be used as the MAX6675 SS
Pin 66 is in Aux2 port adjacent to the Hardware SPI interface port on RAMPS 1.4, and is currently not used for anything.

vandarin committed:
Due to the fix for #1248, X and Y probe offsets must not be floats.

The compiler does not support comparing float values: "error: floating constant in preprocessor expression"
The loss in X/Y precision shouldn't matter for Z probes, as most microswitches or inductive sensors are larger than 1mm square anyway.

drf5n committed:
temperature.cpp: Fix typo of MAX_BED_PID to MAX_BED_POWER.

drf5n committed:
temperature.cpp: Fix typo of PID_MAX to MAX_BED_POWER for the bed.

drf5n committed:
temperature.cpp: Fix typo of PID_MAX to MAX_BED_POWER for the bed.

alhirzel committed:
Fix typo, PID_BED_POWER -> MAX_BED_POWER

alhirzel committed:
Fix typo, PID_BED_POWER -> MAX_BED_POWER

alhirzel committed:
rework analogInputToDigitalPin definition

thinkyhead committed:
Move scripts and documentation to subfolders

monkeydave committed:
Added .gitignore back into repo

monkeydave committed:
Readd .travis.yml, change build enviroment to use ino instead of gcc

monkeydave committed:
Updated .travis.yml for ErikZalm branch pull request

boelle committed:
Update .gitignore

Commit:
temp runaway: proper cast to prevent any overflow.

monkeydave committed:
Updated .travis.yml so others forks will also compile

monkeydave committed:
Added compile checks for lots of defines in Configuration.h

wgm4321 committed:
Fix for cant compile when REPRAP_DISCOUNT_FULL_GRAPHIC_SMART_CONTROLLER defined

wgm4321 committed:
Add English defines for filament menu to non-English to prevent compile errors until translated

monkeydave committed:
Commented out configs that are failing with no easy fixes for the moment

i-make-robots committed:
issue 968 for dev branch

try, try again.

If 'OK' is garbled on sending PC won't receive it.  Both machines will
wait on each other forever.  This resends OK if nothing is heard from PC
for a while to avoid this bad case.

wgm4321 committed:
Add "Detailed Z Probe" menu item with ABL is defined.

wgm4321 committed:
Make sure  volumetric multipliers are recalc'ed during menu enable/disable of mm3

wgm4321 committed:
Revert "Add "Detailed Z Probe" menu item with ABL is defined."

This reverts commit 2b9722f38713bbe4bb664013fa157e5a32ec3e16.

wgm4321 committed:
Revert "Make sure  volumetric multipliers are recalc'ed during menu enable/disable of mm3"

This reverts commit dd7e4ee97f69c835aa26b92fa42056502bd3eac9.

wgm4321 committed:
Disable volumetric menu item due to missing recalc of of multipliers when enabled/disabled

wgm4321 committed:
Re-enable filament menu with recalc fix when enabling/disabling mm3

stv0g committed:
cosmetic whitespace cleanup and comments added/updated

stv0g committed:
improved output aligment and precision

stv0g committed:
more cleanups in the termistorTable generation script

stv0g committed:
refactored termistor table calculation to be in line with wikipedia's article about Steinhart-Hart coefficients

stv0g committed:
more cleanups

stv0g committed:
automatically determine min/max temp for terminator

stv0g committed:
make use of power operator

stv0g committed:
fix #1305 by proposed workaround (thanks gaby64)

stv0g committed:
rename stepper lookup table scipt to be consistent with termistor lookup table script

stv0g committed:
replaced ASCII GPL v3 license by markdown version
moved new COPYING file to top-level directory to make it more visible and remove clutter from Marlin subduer

boelle committed:
added irc channel

boelle committed:
Update README.md

boelle committed:
Update README.md

boelle committed:
Update README.md

boelle committed:
Update README.md

boelle committed:
Update README.md

boelle committed:
Update README.md

DinoMesina committed:
Update Configuration.h

DinoMesina committed:
I have added some code in the "temperature.cpp" for have the possibility of control the time of the states with the heater drived by relays, with this feature now it is possible use PID function in conjunction with relay to control the temperature. I have made some tests and the temperature stay always in the range of +-1°C from the setted temp.
There is the possibility of turn on this features adding "#define SLOW_PWM_HEATERS" in Configuration.h

alexborro committed:
Fix CoreXY speed calculation

For cartesian bots, the X_AXIS is the real X movement and same for
Y_AXIS.
But for corexy bots, that is not true. The "X_AXIS" and "Y_AXIS" motors
(that should be named to A_AXIS
and B_AXIS) cannot be used for X and Y length, because A=X+Y and B=X-Y.
So we need to create other 2 "AXIS", named X_HEAD and Y_HEAD, meaning
the real displacement of the Head.
Having the real displacement of the head, we can calculate the total
movement length and apply the desired speed.

boelle committed:
changed some stuff near IDE

boelle committed:
removed note on Ramps

ramps do have power for the servos, the power comes from the power connector, not sure why this note was even made

boelle committed:
Update README.md

boelle committed:
Update README.md

DinoMesina committed:
Correct some mistakes

boelle committed:
Update README.md

avluis committed:
Update Marlin Firmware URL

Commit:
Added marlin logos. (Designed by Ahmet Cem TURAN)

boelle committed:
added square png version of logo

406*406 pixels

boelle committed:
changed reprap logo to marlin logo

boelle committed:
added link for pic to hex converter

stv0g committed:
moved/renamed documentation stuff to top-level directory

boelle committed:
added link to google hangout

andrewsil1 committed:
Fix grammar/typos in new Configuration.h option

(For the SLOW_PWM_HEATERS option)

ErikZalm committed:
Revert "issue 968 for dev branch"

thinkyhead committed:
Show the latest README on the default branch

thinkyhead committed:
Add documentation logo graphics to Release

AnHardt committed:
Add Version in Configuration.h for Release-branch

thinkyhead committed:
More active authors

thinkyhead committed:
Hardtung

thinkyhead committed:
More authors

YanMolinos committed:
Update language_pt.h

I've changed a few words to better comprehension.

thinkyhead committed:
Notes about the Release branch in its README.md

thinkyhead committed:
Languages of devs not intended as links

AnHardt committed:
Make the Version Numbers for 1.0.2 identical

This is the minimal possible change.
Going over the Configuration.h would require to change them all.

AnHardt committed:
Make the Version Numbers for 1.0.2 identical

This is the minimal possible change.
Going over the Configuration.h would require to change them all.

Wackerbarth commented:
You also missed this line in languages.h

#define MSG_M115_REPORT                     "FIRMWARE_NAME:Marlin V1; Sprinter/grbl mashup for gen6 FIRMWARE_URL:" FIRMWARE_URL " PROTOCOL_VERSION:" PROTOCOL_VERSION " MACHINE_TYPE:" MACHINE_NAME " EXTRUDER_COUNT:" STRINGIFY(EXTRUDERS) " UUID:" MACHINE_UUID "\n"

AnHardt committed:
Make the Version Numbers for 1.0.1 identical

This is the minimal possible change.
Going over the Configuration.h would require to change them all.

AnHardt committed:
Change Version for MSG_M115_REPORT to 1.0.2

Thanks to @Wackerbarth

AnHardt committed:
Change Version for MSG_M115_REPORT to 1.0.2

Thanks to @Wackerbarth

AnHardt commented:
I have an other one ready for 1.0.1
But committing this to 'Release' will result in a 1.0.2 with 1.0.1 Version-Strings.
If we want to do this, we need a further branch.
Anyone a better idea?

Wackerbarth commented:
I suggest that you rebase the release branch (before anyone is really locked into these untested releases.  In the long run, I think that you will be better off. Better to inconvenience a few for a short time than to have a legacy mistake hanging over your head for a long time.

AnHardt committed:
V1.0.1 for M115

AnHardt committed:
Merge remote-tracking branch 'origin/Release' into Release
